src/Okta.Sdk/ApplicationsClient.cs

```diff
@@ -57,6 +58,88 @@ public Task<IApplication> CreateApplicationAsync(CreateBasicAuthApplicationOptio
             return CreateApplicationAsync(app, basicAuthApplicationOptions.Activate, cancellationToken);
         }
 
+        /// <inheritdoc />
+        public async Task<IJsonWebKey> PublishCerCertAsync(string base64EncodedCertificateData, string appId, string csrId, CancellationToken cancellationToken = default(CancellationToken))
+            => await PostAsync<JsonWebKey>(
+                new HttpRequest
+                {
+                    Uri = "/api/v1/apps/{appId}/credentials/csrs/{csrId}/lifecycle/publish",
+                    Verb = HttpVerb.Post,
+                    Payload = Convert.FromBase64String(base64EncodedCertificateData),
```

Comment:
Sanity check - Is this conversion necessary? This request sample seems to be passing a string: https://developer.okta.com/docs/reference/api/apps/#request-example-45

Reply:
I may have misunderstood the requirements of the api.  I'll revisit.

Reply:
Corrected implementation.